tests/revwalk/hidecb.c

```diff
@@ -123,7 +123,46 @@ void test_revwalk_hidecb__add_hide_cb_multiple_times(void)
 
 	cl_git_pass(git_revwalk_new(&walk, _repo));
 	cl_git_pass(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
-	cl_git_fail(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
+	cl_git_pass(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
+	cl_git_pass(git_revwalk_add_hide_cb(walk, NULL, NULL));
+
+	git_revwalk_free(walk);
+}
+
+void test_revwalk_hidecb__unset_cb_before_walk(void)
```

Comment:
This test and the previous one are actually exactly the same, except for that this new one verifies that we _did_ actually unset the hide callback. So you should probably just delete the old test

Reply:
Yep, updated.